DOCSP-34887: v1.11.1 release (#151)
mongodb/docs-kafka-connector: source/whats-new.txt

* :ref:`Version 1.1 <kafka-connector-whats-new-1.1>`
* :ref:`Version 1.0 <kafka-connector-whats-new-1.0>`

.. _kafka-connector-whats-new-1.11.1:

What's New in 1.11.1
--------------------

> - Fixed wildcard matching for partial field name matching. To learn more about

Reviewer: S: The wording of this first sentence is a little confusing to, but possibly because I don't know these features well. Would it make sense to say something like this to avoid the repeated "<feature> matching" phrase:

```suggestion
- Fixed partial name matching with wildcards. To learn more about
```

Author: I think the "with" association makes it less clear how wildcards are associated with partial name matching.

Maybe "Fixed wildcard matching on partial field names in documents." could make this more clear without repeating "matching"?

--------------------

.. important:: Upgrade to Version 1.10.1

  Version 1.9 introduced a bug related to ``MongoSourceTask.start`` that can cause a resource leak on

>   both the connector side and the server side .

Reviewer: ```suggestion
  both the connector side and the server side.
```
